src/components/tool_tip/tool_tip.tsx

```diff
@@ -47,6 +47,13 @@ const positionsToClassNameMap: { [key in ToolTipPositions]: string } = {
 
 export const POSITIONS = keysOf(positionsToClassNameMap);
 
+const displayToClassNameMap = {
+  inlineBlock: undefined,
+  block: 'euiPopover--displayBlock',
```

Comment:
In EUI, we try to keep our class names specific to the component they are in. So we don't want to inherit class names from other components. Therefore, you'll also need to change this and create a new style in the associated SASS file with this class to change the display to block.

```suggestion
  block: 'euiToolTip--displayBlock',
```

Reply:
Whoops! thanks for catching this.

must have copied Popover implementation too much